src/main/java/frc/robot/subsystems/scoring/ScoringSubsystem.java

```diff
@@ -97,6 +98,8 @@ public ScoringSubsystem(
         aimerInterpolated =
                 new InterpolateDouble(
                         ScoringConstants.getAimerMap(), 0.0, ScoringConstants.aimMaxAngleRadians);
+
+        timeToPutAimDown = new InterpolateDouble(ScoringConstants.timeToPutAimDownMap(), 0.0, 2.0);
```

Comment:
Just FYI: I'm not sure that InterpolateDouble will have expected behavior when the angle is out of range: opened #79 for @aidnem to work. For now no action is needed